Program.cs

```diff
@@ -0,0 +1,61 @@
+
+
+namespace Lesson_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            Console.WriteLine("What is your name and surname?");
+            string name1 = Console.ReadLine();
+            Console.WriteLine("How old are you?");
+            string age1 = Console.ReadLine();
+            Console.WriteLine("How much do you weigh?");
+            double weight1 = double.Parse(Console.ReadLine());
+            Console.WriteLine("How tall are you? Answer in CM.");
+            double height1 = double.Parse(Console.ReadLine());
+
+            double bmi1 = weight1 / ( (height1 / 100.0) * (height1 / 100.0) );

```

Comment:
You could divide the variable height1 before: `height1=height1 / 100` to avoid double division or divide one time by 10000 `height1 * height1 / 10000.0`. But it is fine for now I think.

Reply:
Don't divide height1 from 10000, do it on bmi. If you do it for height1, your height1 is not a height anymore

Reply:
@Almantask Is right. This is what I meant, but didn't express myself well enough.